pkgs/top-level/all-packages.nix

```diff
@@ -5186,6 +5186,7 @@ with pkgs;
   };
 
   smarty3 = callPackage ../development/libraries/smarty3 { };
+  smarty3-i18n = callPackage ../development/libraries/smarty3-18n { };
```

Comment:
`s/smarty3-18n/smarty3-i18n/` to fix eval error

Reply:
Thank you! I didn't see that